src/components/organisms/footer/footer.scss

```
@@ -0,0 +1,125 @@
+@use "../../variables" as *;
+
+footer {
+  width: 100%;
+
+  & > .footer-1 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 52px 0;
+    background-color: $white;
+    & > img {
+      height: 25.7px;
+      margin-bottom: 20px;
+    }
+
+    & > p {
+      margin: 0;
+      text-align: center;
+      font-size: 14px;
+      margin-bottom: 20px;
+
+      & > span {
+        margin-right: 14px;
+      }
+    }
+    & > .icon-list {
+      display: flex;
+      width: 98px;
```

```suggestion
      display: flex;
      gap: 14px;
```
こうすると要素間の余白をスマートに指定できるのでおすすめです。